internal/pkg/v2/infrastructure/redis/device_service.go

```diff
@@ -36,9 +50,11 @@ func addDeviceService(conn redis.Conn, ds model.DeviceService) (addedDeviceServi
 		return addedDeviceService, errors.NewCommonEdgeX(errors.KindDuplicateName, fmt.Sprintf("device service name %s already exists", ds.Name), err)
 	}
 
+	ts := common.MakeTimestamp()
```

Comment:
add comment to explain why you need to check the created == 0

Reply:
Added comment.